### DISCUSSION TITLE
should i run one instance of ferretdb for every instance of mongodb i have?

### DISCUSSION THREAD

**Discussion by ghost** [FerretDB/FerretDB#4579] [General]
if i have two apps that leverage a mongodb, can i use one instance of ferretdb to replace mongo and it will manage the separate dbs itself within postgres, or would i theoretically need two instances of ferretdb?

**Comment by AlekSi** [comment]
FerretDB v1 [maps PostgreSQL schemas to MongoDB databases](https://docs.ferretdb.io/understanding-ferretdb/#postgresql), so one PostgreSQL database is enough.
FerretDB v2 (see #4578) has different mapping but also uses a single database.
